bloggies_backend/src/models/email.ts

```diff
@@ -3,6 +3,8 @@ import { MailDataRequired } from '@sendgrid/helpers/classes/mail';
 import ExpressError from "../expressError";
 import { SENDGRID_API_KEY } from '../config';
 
+const frontendUrl = "http://localhost:3000/"; //use env variables to move this to hosted site when applicable
```

Comment:
Very smol thing! Could you capitalize the entire variable to `FRONTEND_URL`? It'll keep the code style consistent with the code base. It also sounds like it's a variable fit to be in the `config.ts` file, so I think moving it there sounds good too! Lmk what you think :D

Reply:
Yes, good idea! Just updated!